src/Middleware/Implementation/DelayMiddlewareInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Queue\Middleware\Implementation;
+
+use Yiisoft\Yii\Queue\Middleware\Push\MiddlewarePushInterface;
+
+/**
+ * A middleware interface for message delaying. It must be implemented in an adapter package or in a project.
+ */
+interface DelayMiddlewareInterface extends MiddlewarePushInterface
```

Comment:
Why do I need this interface?

Reply:
To be able to inject its implementation via DI into the `ExponentialDelayStrategy`